components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -738,8 +739,59 @@ struct LiFiEVMTransaction {
   string gas_limit;
 };
 
+enum LifiToolErrorCode {
+  // No route was found for this action.
+  NO_POSSIBLE_ROUTE = 0,
+  // The tool's liquidity is insufficient.
+  INSUFFICIENT_LIQUIDITY = 1,
+  // The third-party tool timed out.
+  TOOL_TIMEOUT = 2,
+  // An unknown error occurred.
+  UNKNOWN_ERROR = 3,
+  // There was a problem getting on-chain data. Please try again later.
+  RPC_ERROR = 3,
+  // he initial amount is too low to transfer using this tool.
+  AMOUNT_TOO_LOW = 4,
+  // The initial amount is too high to transfer using this tool.
+  AMOUNT_TOO_HIGH = 5,
+  // The fees are higher than the initial amount.
+  // This would result in negative resulting token.
+  FEES_HGHER_THAN_AMOUNT = 6,
+  // This tool does not support different recipient addresses.
+  DIFFERENT_RECIPIENT_NOT_SUPPORTED = 7,
+  // The third-party tool returned an error.
+  TOOL_SPECIFIC_ERROR = 8,
+  // The tool cannot guarantee that the minimum amount will be met.
+  CANNOT_GUARANTEE_MIN_AMOUNT = 9
+};
+
+// https://github.com/lifinance/types/blob/main/src/errors.ts
+enum LiFiErrorCode {
+  Success = 0,
+  DefaultError = 1000,
+  FailedToBuildTransactionError = 1001,
+  NoQuoteError = 1002,
+  NotFoundError = 1003,
+  NotProcessableError = 1004,
+  RateLimitError = 1005,
+  ServerError = 1006,
+  SlippageError = 1007,
+  ThirdPartyError = 1008,
+  TimeoutError = 1009,
+  UnauthorizedError = 1010,
+  ValidationError = 1011,
+};
```

Comment:
Guide requires us to use kPrefxied convention.
So `LifiToolErrorCode.kNoPossbileRoute`, `LiFiErrorCode.kDefaultError` and so on
https://chromium.googlesource.com/chromium/src/+/master/mojo/public/tools/bindings/README.md#enumeration-types

Reply:
fixed